src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1407,12 +1400,10 @@ internal async ValueTask<NpgsqlDataReader> ExecuteReader(CommandBehavior behavio
                 throw;
             }
 
-            async Task NonMultiplexingWriteWrapper(NpgsqlConnector connector, bool async, CancellationToken cancellationToken2)
+            Task NonMultiplexingWriteWrapper(NpgsqlConnector connector, bool async, CancellationToken cancellationToken)
```

Comment:
This can now be inlined back into the call side (:tada). The only reason it was a different function is to have an async method returning a Task representing both write and flush, but Write now does that.